core/neat/mixins/_grid-media.scss

```diff
@@ -12,7 +19,13 @@
 ///   The grid to be used within the scope of the block.
 ///   This grid should include the `media` property to determine the expression
 ///   for the media query.
-///   By default, the global `$neat-grid` will be used.
+///
+/// @content
+///   Grid media will temporarily the override the default grid with the
+///   attributes of the custom grid. This change is scoped to within the mixin
+///   block, and once the mixin has ended the default grid will revert to its
+///   original state as defined by `$neat-grid`. This allows for different
+///   gutter width and column count based on screen size or other properties.
 ///
```

Comment:
I think further clarifying the custom neat properties here will be extremely helpful. i.e.:
```
/// @property {number (unitless)} columns [12]
///   Default number of the total grid columns.
///
/// @property {number (with unit)} gutter [20px]
///   Default grid gutter width between columns.
///
/// @property {number (with unit) | string} media [null]
///   Grid media query. If only a length with unit is passed (such as 280px), `grid-media` assumes min-width. 
///   Otherwise, `grid-media` will pass the entire string along to the media query. For example:
///   `'only screen and (min-width: 480px) and (max-width: 520px)'`
///
/// @property {color} color [rgba(#00d4ff, 0.25)]
///   Default visual grid color.
```